### ISSUE TITLE
Initials matching a conjunction cause an unparsable error

### ISSUE THREAD

**Issue by GoogleCodeExporter** [gdsaxton/python-nameparser#11]
```
HumanName('E.T. Smith')
ERROR:HumanName:Unparsable full_name: E.T. Smith

Because 'et' is a conjunction.  Again, removing periods indiscriminately loses 
information.
```

Original issue reported on code.google.com by `jayqhac...@gmail.com` on 8 Feb 2012 at 3:35

**Comment by GoogleCodeExporter** [comment]
Original comment by `dere...@gmail.com` on 7 Oct 2012 at 10:52
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
a further problem: A name containing the middle initial 'e' will not parse 
correctly. e.g.: 'john e smith' parses out to first_name: 'john e smith', 
middle name and last name are blank.

```

Original comment by `randy.ed...@gmail.com` on 4 Dec 2012 at 10:30

**Comment by GoogleCodeExporter** [comment]
```
having 'e' in the list of conjunctions causes any name with the first initial 
'e' to become unparseable. 'a j smith' parses correctly, 'e j smith' does not.
```

Original comment by `randy.ed...@gmail.com` on 4 Dec 2012 at 10:32

**Comment by GoogleCodeExporter** [comment]
```
This should be fixed in v0.2.4
```

Original comment by `dere...@gmail.com` on 11 Feb 2013 at 3:48
- Changed state: **Fixed**
